test/packages/good/elasticsearch/ml_model/good-ml-model-abc-1.json

```diff
@@ -0,0 +1,99 @@
+{
+  "model_id": "default",
```

Comment:
Just to be sure: this file will be installed in Elasticsearch by Kibana, right? If so, could you please confirm with @jen-huang that this structure is valid?